tools/vsimporter/xib2nib/UIControl.cpp

```diff
@@ -35,6 +36,39 @@ void UIControl::InitFromXIB(XIBObject *obj)
 void UIControl::InitFromStory(XIBObject *obj)
 {
     UIView::InitFromStory(obj);
+
+    const char* horizontalAlign = getAttrib("contentHorizontalAlignment");
+    if (horizontalAlign) {
+        if (strcmp(horizontalAlign, "center") == 0) {
+            _contentHorizontalAlignment = (int)UIControlContentHorizontalAlignmentCenter;
+        } else if (strcmp(horizontalAlign, "left") == 0) {
+            _contentHorizontalAlignment = (int)UIControlContentHorizontalAlignmentLeft;
+        } else if (strcmp(horizontalAlign, "right") == 0) {
+            _contentHorizontalAlignment = (int)UIControlContentHorizontalAlignmentRight;
+        } else if (strcmp(horizontalAlign, "fill") == 0) {
+            _contentHorizontalAlignment = (int)UIControlContentHorizontalAlignmentFill;
+        } else {
+            assert(0);
+        }
+        getAttrAndHandle("contentHorizontalAlignment");
```

Comment:
Issue getAttrAndHandle (line 40) in the initial call and then remove this line. Do the same for verticalAlign.